api/data_explorer/util/elasticsearch_util.py

```diff
@@ -164,6 +164,25 @@ def get_facet_value_dict(filter_arr, facets):
     return parsed_filter
 
 
+def get_field_description(es, field_name):
+    s = Search(using=es, index=current_app.config['INDEX_NAME'] + '_fields')
+    s.update_from_dict({
```

Comment:
Is it possible to use elasticsearch-dsl? Then it would be easier to maintain, one wouldn't have to deal with lining up brackets.

Reply:
Hmm, Search() is from elasticsearch-dsl. I wasn't able to figure out how to set the query with elasticsearch-dsl.